src/test/java/testNG/TestReg.java

```diff
@@ -0,0 +1,12 @@
+package testNG;
+
+import holders.TestInit;
+import org.testng.annotations.Test;
+
+public class TestReg extends TestInit {
```

Comment:
The same for this class, name isn’t clear

Reply:
ok

Reply:
Thats name of test , didnt understand